lib/rucio/db/sqla/models.py

```diff
@@ -752,11 +752,6 @@ class RSE(BASE, SoftModelBase):
                       default=False)
     staging_area = Column(Boolean(name='RSE_STAGING_AREA_CHK', create_constraint=True),
                           default=False)
-    city = Column(String(255))
-    region_code = Column(String(2))
-    country_name = Column(String(255))
-    continent = Column(String(2))
-    time_zone = Column(String(255))
     ISP = Column(String(255))
```

Comment:
i think these two (ISP and ASN) can go as well

Reply:
Erased from existence